### PR TITLE
1st chunk for German translation

### PR DESCRIPTION
**Pull request by boris779**
I added all the necessary files and settings for German translation.
Further all headlines/menu entries are now in German.

I will translate chapter by chapter

### CODE REVIEW

File: docs_source_files/content/CONTRIBUTING.de.md [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+% git fetch upstream
+% git rebase upstream/master
+```
+
+### Step 6: Test
+
+Always remember to [run the local server](https://gohugo.io/getting-started/usage/#livereload),
+with this you can be safe that your changes have not broken anything.
+
+### Step 7: Sign the CLA
```
Comment (diemol) [inline docs_source_files/content/CONTRIBUTING.de.md]: This file is out of date, for example, we do not have this step anymore.

File: docs_source_files/content/findAndReplaceText.sh [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# find and copy:
```
Comment (diemol) [inline docs_source_files/content/findAndReplaceText.sh]: What is this file? Did it get committed by mistake?

File: docs_source_files/content/remote_webdriver/remote_webdriver_client.de.md [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+driver.get("http://www.google.com")
+driver.quit()
+  {{< / code-panel >}}
+{{< / code-tab >}}
+
+
+To further customize our test configuration, we can add other desired capabilities.
+
+
+## Desired capabilities
```
Comment (diemol) [inline docs_source_files/content/remote_webdriver/remote_webdriver_client.de.md]: This is also out of date. I believe the fork was not updated with the recent changes.

**Comment by boris779** [comment]
Hi @diemol! Sorry my fault, it took me too long to push this Pull request. So I thought this is a small chunk because only headlines and navigation are translated. 

So what do you think is the best way without losing my changes so far? I don't want to start over. Minutes ago I merged everything from upstream/master into my german branch. 
Now I have to compare all the *.de.md files with the en.md files right?

**Comment by boris779** [comment]
So before this happens again, I updated all german files by manual comparison via diff in my IDE. 

Hopefully, this first (big) chunk can be verified. Further translations will be of course in small chunks (page or topics).

File: docs_source_files/content/findAndReplaceText.sh (hunk 2 above)
Comment (boris779) [reply docs_source_files/content/findAndReplaceText.sh]: sorry. was a script to create all the de files, I removed it already

File: docs_source_files/content/remote_webdriver/remote_webdriver_client.de.md (hunk 3 above)
Comment (boris779) [reply docs_source_files/content/remote_webdriver/remote_webdriver_client.de.md]: I updated my PR, see comment below.

**Comment by diemol** [comment]
@boris779 have you seen that the build is failing?

**Comment by boris779** [comment]
Sorry missed that.

**Comment by boris779** [comment]
now also updated german files from english source.

**Pull request merged**